src/util/integerfactoring.c

```diff
@@ -698,22 +1253,145 @@ static uint64_t gaIFLGetSmallestFactorv(int n, const ga_factor_list* fl, int* id
 	return hasFactors ? f : 1;
 }
 
+int      gaIFLsprintf(char* str, const ga_factor_list* fl){
```

Comment:
Yes I did miss that.  However, I wonder why you need an sprintf-like function.  If it is only for debugging, I would prefer to go with append() only.